salesforce.go

```diff
@@ -603,5 +603,8 @@ func (sf *Salesforce) GetJobResults(bulkJobId string) (BulkJobResults, error) {
 }
 
 func (sf *Salesforce) GetAccessToken() string {
-	return sf.auth.AccessToken
+	if sf.auth != nil {
```

Comment:
```go
        if sf.auth == nil { 
                 return ""
         } 
         return sf.auth.AccessToken
```

Usually it's written like this for clarity purpose

Reply:
@ccoVeille Actually, that's a good point. I'll make that update.